core/src/main/java/org/keycloak/protocol/oidc/representations/OIDCConfigurationRepresentation.java

```diff
@@ -142,8 +142,30 @@
     @JsonProperty("backchannel_logout_session_supported")
     private Boolean backchannelLogoutSessionSupported;
 
+    @JsonProperty("grant_id_supported")
+    private Boolean grantIdSupported;
```

Comment:
According to https://openid.net/specs/fapi-grant-management-01.html#section-5.1 , it might be "none", "optional" or "always".
As mentioned later, "optional" might be appropriate.